### ISSUE TITLE
No data coming in, everything seems fine. 

### ISSUE THREAD

**Issue by butterflyology** [0RuiAlvel0/Airport-departure-board#8]
I am new to this type of project and feel that I missed something simple, but I don't know what I don't know.

I followed the instructions and feel that everything is set up correctly, but I see no data populate my departure board when I navigate to base.php. The airport I selected is correctly displayed, only no flights. 

- I have the api key and confirmed it works (with the 2k/month limit) with airlabs. The api key is in `api.php`
- The usage tab at airlabs shows queries have been made
- The `error.log` only shows restarts of Apache2
- Ownership of `www-data` has been changed and the airport I selected appears on the website for the program (following the `README` directions)
- My normal user is in `www-data` (following the `README` directions)

I would greatly appreciate any help troubleshooting this.

**Comment by robsonma1984** [comment]
Yes I can also confirm this does not work, this is such a shame as it is an amazing tool but so sad this does not work.
